string node_name
string node_class
---
